ImperatorToCK3/Data_Files/configurables/replaceable_file_blocks.txt

```diff
@@ -110,6 +110,2060 @@
+		}
+
+		root = {
+			set_primary_title_to = title:e_western_roman_empire
+		}
+
+		resolve_title_and_vassal_change = scope:change
+
+		root = {
+			destroy_title = title:e_roman_empire
+		}
+		} # end of after
+	}
+
+}
+
+
+"common/character_interactions/00_character_interactions.txt" = {
+	
+	# offer_vassalization_interaction ai_accept block
+	replace = {
+		before = {
+					OR = {
+						this = culture:greek
+						any_parent_culture_or_above = {
+							this = culture:greek
+						}
+						this = culture:han
+						any_parent_culture_or_above = {
+							this = culture:han
+						}
+					}
+		}# end of before
+
+		after = {
+					OR = { # IRToCK3: Modified to include roman since this is historical admin realm cultures
+						this = culture:greek
+						this = culture:roman
+						any_parent_culture_or_above = {
+							this = culture:greek
+						}
+						any_parent_culture_or_above = {
+							this = culture:roman
+						}
+						this = culture:han
+						any_parent_culture_or_above = {
+							this = culture:han
+						}
+					}
+		} # end of after
+	}
+
+}
+
+
+"common/culture/traditions/07_ep3_traditions.txt" = {
+	
+	# tradition_ep3_imperial_tagmata is_shown block
+	replace = {
+		before = {
+		OR = {
+			this = culture:greek
+			any_parent_culture_or_above = {
+				this = culture:greek
+			}
+			has_cultural_pillar = heritage_byzantine
+		}
+		# DLC check.
+		has_ep3_dlc_trigger = yes
+		NOT = {
+			has_cultural_tradition = tradition_roman_legacy
+		}
+		}# end of before
+
+		after = {
+		OR = { # IRToCK3: Added roman as possible culture
+			this = culture:greek
+			this = culture:roman
+			any_parent_culture_or_above = {
+				this = culture:greek
+			}
+			any_parent_culture_or_above = {
+				this = culture:roman
+			}
+			#has_cultural_pillar = heritage_byzantine # IRToCK3: Removed since this heritage know represents general Hellenistic cultures, so it doesn't really fit
+		}
+		# DLC check.
+		has_ep3_dlc_trigger = yes
+		NOT = {
+			has_cultural_tradition = tradition_roman_legacy
+		}
+		} # end of after
+	}
+	
+	# tradition_ep3_roman_ceremonies & tradition_ep3_palace_politics is_shown block
+	replace = {
+		before = {
+		OR = {
+			this = culture:greek
+			any_parent_culture_or_above = {
+				this = culture:greek
+			}
+			has_cultural_pillar = heritage_byzantine
+			scope:character = {
+				OR = {
+					has_title = title:e_byzantium
+					has_title = title:e_roman_empire
+				}
+			}
+		}
+		# DLC check.
+		has_ep3_dlc_trigger = yes
+		NOT = {
+			has_cultural_tradition = tradition_byzantine_succession
+		}
+		}# end of before
+
+		after = {
+		OR = { # IRToCK3: Added roman as possible culture
+			this = culture:greek
+			this = culture:roman
+			any_parent_culture_or_above = {
+				this = culture:greek
+			}
+			any_parent_culture_or_above = {
+				this = culture:roman
+			}
+			#has_cultural_pillar = heritage_byzantine # IRToCK3: Removed since this heritage know represents general Hellenistic cultures, so it doesn't really fit
```

Comment:
```suggestion
			#has_cultural_pillar = heritage_byzantine # IRToCK3: Removed since this heritage now represents general Hellenistic cultures, so it doesn't really fit
```